_schemaV2.graphql

```diff
@@ -6987,6 +6987,7 @@ type Me implements Node {
     state: CommerceOrderStateEnum
   ): CommerceOrderConnectionWithTotalCount
   paddleNumber: String
+  pendingIdentityVerificationId: String
```

Comment:
In general we'd like to keep client logic as simple as possible. I think providing the URL (so that every client doesn't need to manually construct it) would be a good addition. It also makes it where we only need to go to one place to make an updated. 

You could also just have an `IndentityVerification` or `PendingIdentityVerification` object that contains both the id and the URL so that either are retrievable if necessary. If you think that the id would only be used for generating the URL though, then only passing the URL would be good.